### PR TITLE
u/parejkoj/multiplier-docstring: Clarify multiplier docstring

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

File: python/lsst/meas/algorithms/detection.py [hunk 1]
```diff
@@ -64,11 +64,13 @@ class SourceDetectionConfig(pexConfig.Config):
...
     )
     thresholdValue = pexConfig.RangeField(
-        doc="Threshold for footprints; exact meaning and units depend on thresholdType.",
+        doc="Threshold for detecting footprints; exact meaning and units depend on thresholdType.",
         dtype=float, optional=False, default=5.0, min=0.0,
     )
     includeThresholdMultiplier = pexConfig.RangeField(
-        doc="Include threshold relative to thresholdValue.",
+        doc="Multiplier on thresholdValue for whether a source is included in the output catalog. "
+            "For example, value=5, multiplier=10 results in a catalog of sources at >50 sigma, "
```
Comment (laurenam) [inline python/lsst/meas/algorithms/detection.py:72]: I'd just use the actual config names here, i.e. `value` -> `thresholdValue` and `multiplier` -> `includeThresholdMultiplier`.  Also, I'm not sure `sigma` is strictly correct given the options for `thresholdType`.

**Review by PaulPrice — APPROVED** [review]
Lauren covered it.

File: python/lsst/meas/algorithms/detection.py (hunk 1 above)
Comment (parejkoj) [reply python/lsst/meas/algorithms/detection.py]: How is it now?

Comment (laurenam) [reply python/lsst/meas/algorithms/detection.py]: LGTM

**Review by laurenam — APPROVED** [review]

**Pull request merged**